apps/website/docs/actions/button-group.mdx

```diff
@@ -1,66 +1,124 @@
 ---
 title: Button Group
-description: Group related buttons.
+description: Button groups are designed to bring together buttons that are categorically related to each other.
 sidebar_custom_props:
   status: 'production'
 ---
 
-Group related buttons.
+Button groups are designed to bring together buttons that are categorically related to each other. For example, you'd use a button group to group text formatting controls like Bold, Underline, and Italicize.
 
 ```jsx live
-<ButtonGroup>
-  <Button>Button</Button>
-  <Button>Button</Button>
+<ButtonGroup variant="secondary" isAttached>
+  <Button>Bold</Button>
+  <Button>Underline</Button>
+  <Button>Italicize</Button>
 </ButtonGroup>
 ```
 
-## Usage
+## Appearance
```

Comment:
Hmm I have a concern removing the `Usage` entirely, it is a terminology pattern adopted widely.

Reply:
Totally fair. I do use "Appearance" to describe the different variants on the Buttons doc, too. That felt appropriate since it was describing the different visual styles of the buttons, but I don't feel strongly either way. If "Usage" is a common pattern in DS documentation, happy to follow that.

Reply:
Ah I missed that then. I think that since every component's page follows the hierarchy of Usage and Props, then sub bullets within those, we should either keep Usage as top level or consider a wider overhaul to maintain consistency between components